projects/packages/forms/src/blocks/contact-form/attributes.js

```diff
@@ -52,4 +52,8 @@ export default {
 			listName: null,
 		},
 	},
+	saveResponses: {
+		type: 'string',
+		default: 'yes',
+	},
```

Comment:
I understood there was a shortcodes-related legacy reason these had to be carried around as "yes/no" rather than `true/false`, but was it necessary for the block attributes as well? If in future we refactor away from shortcode-mangled way of doing this, we'll then need deprecation method for the attributes too instead of just refactoring the backend.

I see we use boolean in one of the attributes:

https://github.com/Automattic/jetpack/blob/dda00a325d883a530f3cd816d45990673200addb/projects/packages/forms/src/blocks/contact-form/attributes.js#L28-L31

Basically, is there a way to avoid having the attribute here as a string even if it is later carried around as a string? :-)